ovs/lib/alba.py

```diff
@@ -317,16 +316,24 @@ def _add_backend_osds(alba_backend_guid, osds, domain, metadata):
                                               'alba-osd-config-url': 'file://{0}'.format(remote_arakoon_config)})
                 except AlbaError as ae:
                     AlbaController._logger.exception('Error adding OSD {0}: {1}'.format(linked_alba_id, ae))
-                    failures.append(linked_alba_id)
+                    if ae.error_code == 7 and ae.exception_type == AlbaError.ALBAMGR_EXCEPTION:
+                        AlbaController._logger.warning('OSD with ID {0} has already been added'.format(linked_alba_id))
+                        unclaimed_osds.append(linked_alba_id)
+                        continue
+                    failed_claims.append(linked_alba_id)
```

Comment:
Perhaps also log an exception in this case, similar to the flow in `claim-osd`
Also remove the part where `ae` is being logged there, useless since we log on exception level